docs/how_to/single_node.md

```diff
@@ -159,3 +119,7 @@ curl -sSf -X POST http://localhost:9003/query -d '{
   "exhaustive": true
 }
 ```
+
+## Use integrations
```

Comment:
`Integrations` might be a better header than `Use integrations`

Reply:
Sure thing, done.